### ISSUE TITLE
deconz devices respond with a delay

### ISSUE THREAD

**Issue by vantino** [dresden-elektronik/deconz-rest-plugin#4042]
## Describe the question or issue you are having

Since the kids crashed our pi3/raspbee I with water, I ordered raspbee2.
Installed and runing at a new pi3+ .. all good.

However, from time to time I "enjoy" kind of delay situations, which were never an issue with the older version (raspbee I).

Once, I want to turn a hue light on, it reacts with a delay of 3-5 sec. and sometimes the signal is not being received at the device, but the next signal works.
If my alexa tells the fhem-connector to turn on a light, sometimes nothing happens. However, the next command works immediately.

 pi3 is on wlan connection to the router and i tried several wlan cannels. same issue with a cable connection. And pi3 is far away from any interference device like usb3, ...

## Environment

 - Host system: (Raspberry Pi  3+ Model B)
 - Running method: (Raspbian headless)
 - Firmware version: (26xxyy00)
 - deCONZ version: (2.07.01)
 - Device: (RaspBee II)
 - Do you use an USB extension cable: ( no) -- only relevant for ConBee I/II
 - Is there any other USB or serial devices connected to the host system? No

## Additional context

To be fair, it may be an issue with the fhem service at the raspi.  However, all software parts are the same, as used at the old configuration, the new part is just the raspbee II.

Any idea?

THanks ROland

**Comment by Mimiix** [comment]
What I've seen in cases like these, is that it mostly due to a slow sd card or a overloaded pi.

Or, of course, network issues. That would show in the logs with routes being unavailable.

**Comment by vantino** [comment]
Hm, thanks for reply.

CPU load is constantly 8 -13% and 18% sporadically,
no network issues found
I use SanDisk Ultra 64GB Class 10 
no power issues, like "low voltage" ...

However, will check cpu load next time

**Comment by vantino** [comment]
Thanks for your support!

With more than 12h runtime w/o any issues it looks like to be a network problem, as you suguested initially.  I found massive msg's in kern.log and broadcast msg's in syslog with massive increasing size of both files.

However, the route course is the "Phillips Hue Smart Button", which is the only change in the new system.
I removed its battery and my fhem (13 lights, 3 sensors, 2 switches) works as it did the last 18 month.

Finally - 

Phillips Hue Smart Button interferes Raspi 3+ Mod B "brcmf_cfg80211" module.

CHeers
Roland
